lms/djangoapps/courseware/tests/test_entrance_exam.py

```diff
@@ -332,6 +313,10 @@ def test_entrance_exam_requirement_message_with_correct_percentage(self):
         minimum_score_pct = 29
         self.course.entrance_exam_minimum_score_pct = float(minimum_score_pct) / 100
```

Comment:
I completely agree.  This initially threw me off as well.  I had assumed they were percentages because of the previous code, but they ended up being < 1 (ratios).  That's why I had named the methods as percentages.  But then, in my debugging, I realized the code expected ratios.  I'll go ahead and change the names of at least the new methods - and a comment on the pre-existing field (which is hard to change now).